package.json

```diff
@@ -18,6 +18,7 @@
     "lint": "grunt lint",
     "test": "grunt test",
     "publish-coverage": "grunt coveralls",
+    "nsp-check": "nsp check -o summary",
```

Comment:
Since this is an npm script, you can still do `npm run nsp-check` locally to check for vulnerable dependencies, not limited to just running on CI (and doesn't require a globally installed module).